### DISCUSSION TITLE
call the log-view route with vue

### DISCUSSION THREAD

**Discussion by maxdiable** [opcodesio/log-viewer#62] [Q&A] [answered]
it's possible to call the log-view route with vue?

br
Max

**Comment by arukompas** [comment, marked as answer]
Hey @maxdiable ,

Sorry, but this package currently does not provide a JSON API. You have to use the interface provided by visiting the `/log-viewer` path (or whatever path you've configured in `config/log-viewer.php`).

**Comment by abdosaeedelhassan** [comment]
@maxdiable 
i think you can use the iframe to call the log-viewer page inside vue component
